Description: Add support for steelseries Gaming headphones
Origin: upstream, https://cgit.freedesktop.org/pulseaudio/pulseaudio/commit/?id=c7fe78c9f73ded2c3428666722ec9c1af4b82812
Bug-Ubuntu: https://launchpad.net/bugs/1758736
Author: Dave Chiluk <chiluk@ubuntu.com>

-----------------------------------------------
commit c7fe78c9f73ded2c3428666722ec9c1af4b82812
Author: Tanu Kaskinen <tanuk@iki.fi>
Date:   Sat Sep 2 18:23:12 2017 +0300

    build-sys: add the Arctis configuration

Index: pulseaudio-11.1/src/Makefile.am
===================================================================
--- pulseaudio-11.1.orig/src/Makefile.am
+++ pulseaudio-11.1/src/Makefile.am
@@ -1327,6 +1327,7 @@ dist_alsaprofilesets_DATA = \
 		modules/alsa/mixer/profile-sets/native-instruments-korecontroller.conf \
 		modules/alsa/mixer/profile-sets/kinect-audio.conf \
 		modules/alsa/mixer/profile-sets/sb-omni-surround-5.1.conf \
+		modules/alsa/mixer/profile-sets/steelseries-arctis-usb-audio.conf \
 		modules/alsa/mixer/profile-sets/dell-dock-tb16-usb-audio.conf
 
 if HAVE_UDEV
@@ -1368,7 +1369,10 @@ dist_alsapaths_DATA = \
 		modules/alsa/mixer/paths/hdmi-output-4.conf \
 		modules/alsa/mixer/paths/hdmi-output-5.conf \
 		modules/alsa/mixer/paths/hdmi-output-6.conf \
-		modules/alsa/mixer/paths/hdmi-output-7.conf
+		modules/alsa/mixer/paths/hdmi-output-7.conf \
+		modules/alsa/mixer/paths/steelseries-arctis-input.conf \
+		modules/alsa/mixer/paths/steelseries-arctis-output-mono.conf \
+		modules/alsa/mixer/paths/steelseries-arctis-output-stereo.conf
 
 endif
 
